coalib/coala_main.py

```diff
@@ -63,6 +65,8 @@ def run_coala(log_printer=None,
         dtag = str(sections['default'].get('dtag', None))
         config_file = os.path.abspath(str(sections["default"].get("config")))
 
+        verify_file_integrity(log_printer)
```

Comment:
yeah this should happen automaticlaly when you load the stuff and it just loads an empty dict  if data is corrupt so you can drop this line